# ==============================================================================
# REGRESSOR COMMIT DIFF
# ==============================================================================
# File: gfx/layers/d3d11/CompositorD3D11.h
# Commit: 99b2e5c55a25
# Full Hash: 99b2e5c55a25f47f9a5a0650607cd54ce966c28b
# Author: Matt Woodrow <mwoodrow@mozilla.com>
# Date: 2020-11-05 16:09:03
# Regressor Bug: 1673983
# File Overlap Count: 2
# Description:
#   Bug 1673983 - Add D3D11 RenderCompositor for SWGL. r=lsalzman
#   
#   Differential Revision: https://phabricator.services.mozilla.com/D95797
# ==============================================================================

diff -r 96087b511fe9 -r 99b2e5c55a25 gfx/layers/d3d11/CompositorD3D11.h
--- a/gfx/layers/d3d11/CompositorD3D11.h	Thu Nov 05 00:51:24 2020 +0000
+++ b/gfx/layers/d3d11/CompositorD3D11.h	Thu Nov 05 09:14:52 2020 +0000
@@ -142,6 +142,13 @@
     mAllowFrameRecording = aWillRecord;
   }
 
+  void Readback(gfx::DrawTarget* aDrawTarget) {
+    mTarget = aDrawTarget;
+    mTargetBounds = gfx::IntRect();
+    PaintToTarget();
+    mTarget = nullptr;
+  }
+
  private:
   enum Severity {
     Recoverable,